src/base/datetime/calendar/header/test/language.test.ts

```diff
@@ -0,0 +1,95 @@
+import { expect, fixture } from "@open-wc/testing";
+import { BaseDateTimeCalendarHeader } from "../index";
+
+describe("BaseDateTimeCalendarHeader", () => {
+  describe("language", () => {
+    it("should be 'en' when not assigning on constructor", async () => {
+      const container = new BaseDateTimeCalendarHeader();
+      const el = await fixture(container);
+      const headerCenterEl = el.querySelector(
+        ".kuc-base-datetime-calendar-header__group__center"
+      ) as HTMLSpanElement;
+
+      expect(headerCenterEl.childElementCount).to.equal(2);
+      expect(headerCenterEl.children[0].className?.trim()).to.equal(
```

Comment:
@dong0 
There is the logic `className?` like if/else condition.
Please remove the logic for test.
Ex:
expect(headerCenterEl.children[0].classList.contains('kuc-base-datetime-calendar-header__group__month')).to.equal(true);